blackbox/docs/sql/scalar.txt

```diff
@@ -215,6 +215,41 @@ locale-sensitive or context-sensitive mappings.
     SELECT 1 row in set (... sec)
 
 
+sha1('string') returns string
+-----------------------------
+
+Computes the SHA1 checksum of the given string.
+
+::
+
+    cr> select sha1('crate')
+    ... limit 1;
+    +------------------------------------------+
+    | sha1('crate')                            |
+    +------------------------------------------+
+    | 1673dc397042322a0a5ac49c79cc08d3a25cb0f6 |
+    +------------------------------------------+
+    SELECT 1 row in set (... sec)
+
+sha256('string') returns string
```

Comment:
i am not sure if we should introduce another new function for 512 and 256, mysql has sha2(string, l) which i would prefer here